src/data/AkitaPaymentPointerData.js

```diff
@@ -20,7 +23,10 @@ class AkitaPaymentPointerData {
 	 * @return {AkitaPaymentPointerData} the input object as an instance of the AkitaPaymentPointerData class.
 	 */
 	static fromObject(akitaPaymentPointerDataObject) {
-		const newPaymentPointerData = new AkitaPaymentPointerData(akitaPaymentPointerDataObject.paymentPointer);
+		const newPaymentPointerData = new AkitaPaymentPointerData(
+			akitaPaymentPointerDataObject.paymentPointer
+		);
```

Comment:
nit: I think this can just be a single line. `AkitaPaymentPointerData()` only takes 1 param

Reply:
okay. done.